### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by fernmac2** [gysmo38/mitsubishi2MQTT#210] [Q&A] [answered]
Sorry if this is obvious, I have version 2022.01, is there a way I can upgrade to the latest version, without flashing again?
Thanks

**Comment by kayak01** [comment]
If you connect to the controller via a browser, you can just upload the new version and it will update the flash.  It just takes a few seconds to do.

**Comment by vperas** [comment, marked as answer]
> Sorry if this is obvious, I have version 2022.01, is there a way I can upgrade to the latest version, without flashing again? Thanks

If using Arduino IDE click on "Sketch" then "Export compiled binary". After compiling is finished, find the directory where the *.bin file is (somewhere in ...\Documents\Arduino\\"folder name"\build\...) and upload it from web page of your device via "Firmware upgrade"

![main_page](https://user-images.githubusercontent.com/94906061/223557695-670de8e1-7a9c-4678-83f8-21e4f0972bcc.png)

Hope that helps.

**Reply by fernmac2** [reply]
Thank you so much, exactly what I needed!
